fixtures/docker-buildx-stop.1

```diff
@@ -0,0 +1,33 @@
+.nh
+.TH "DOCKER" "1" "Jan 2020" "Docker Community" "Docker User Manuals"
+
+.SH NAME
+.PP
+docker-buildx-stop - Stop builder instance
+
+
+.SH SYNOPSIS
+.PP
+\fBdocker buildx stop [NAME]\fP
+
+
+.SH DESCRIPTION
+.PP
+Stop builder instance
```

Comment:
Do we need a fixture / example with a long description (taken from markdown)? (I think that's implemented in this PR as well, correct?)

Reply:
Added "attach" and "buildx dial-stdio" that contains long description for testing